app.wxss

```diff
@@ -9,3 +9,7 @@
   box-sizing: border-box;
 } 
 
+
+.page__container {
```

Comment:
Hey this class is unused, why dont you delete it?

Reply:
already deleted, all done!